frontend/javascripts/oxalis/model/sagas/volumetracing_saga.tsx

```diff
@@ -703,9 +710,11 @@ export function* diffVolumeTracing(
       prevVolumeTracing.mappingName !== volumeTracing.mappingName ||
       prevVolumeTracing.mappingIsPinned !== volumeTracing.mappingIsPinned
     ) {
+      // Once the first volume action is performed on a volume layer, the mapping state is pinned.
+      // In case no mapping is active, this is denoted by setting the mapping name to null.
       const action = updateMappingName(
-        volumeTracing.mappingName,
-        volumeTracing.mappingIsEditable,
+        volumeTracing.mappingName || null,
+        volumeTracing.mappingIsEditable || null,
```

Comment:
Are the `|| null` needed, because the backend expects `null` instead of `undefined` in that case?